dev/com.ibm.ws.security.common.jsonwebkey/src/com/ibm/ws/security/common/jwk/impl/JwKRetriever.java

```diff
@@ -439,7 +449,7 @@ protected Key doJwkRemote(String kid, String x5t, String use, boolean useSystemP
                 sslSocketFactory = getSSLSocketFactory(locationUsed, sslConfigurationName, sslSupport);
             }
             HttpClient client = createHTTPClient(sslSocketFactory, locationUsed, hostNameVerificationEnabled, useSystemPropertiesForHttpClientConnections);
-            jsonString = getHTTPRequestAsString(client, locationUsed);
+            jsonString = httpUtils.getHttpJsonRequest(client, locationUsed);
```

Comment:
We'll need to catch exception(s) from getHttpJsonRequest(...) and preserve existing behavior when tracing the errors with logCWWKS6049E(...)